README.md

```diff
@@ -83,3 +83,5 @@ Example user story: `As an admin I want to be able to create new users because I
 5. Delete the 'in-progress' label
 6. Update the time label if neccessary
 7. Close the user story referencing the PR
+
+_These guidelines for creating user stories are an adaptation of [Simon's](https://github.com/simonlab) [**user-story-checklist**](https://github.com/SimonLab/user-story-checklist) repo_
```

Comment:
👍 
I've _often_ thought of sending Simon a PR expanding his checklist... finding the time...!! ⌛